packages/core/src/data-module/data-cache/types.ts

```diff
@@ -17,6 +18,7 @@ export type HistoricalRequest = {
 export type DataStreamStore = {
   id: string;
   resolution: number;
+  aggregationType: AggregateType;
```

Comment:
I think aggregationType should be conditional since we don't need it for raw data.